app/src/main/java/bubbleheads/buy_cook/Category.java

```diff
@@ -1,19 +1,27 @@
 package bubbleheads.buy_cook;
 
-public enum Category {
-
-    SOUPS("Soups"), PIZZA("Pizza"), MEAT("Meat"), SALADS("Salads"),
-    LUNCH("Lunch"), MAIN_DISHES("Main dishes"), DRINKS("Drinks"),
-    DESSERTS("Desserts"), COOKIES("Cookies"), CAKES("Cakes"),
-    PIES("Pies"), VEGAN("Vegan"), VEGETERIAN("Vegeterian");
+public class Category {
 
     private final String name;
+    private final int categoryID;
+    private static int globalID = 0;
 
     Category(final String name) {
         this.name = name;
+        globalID++;
+        this.categoryID = globalID;
```

Comment:
you could just write this.categoryID = globalID++ or this.categoryID = ++globalID depending on whether or not zero is considered a valid ID